pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -273,3 +280,59 @@ func isMlnxNicAndInNode(pciAddress string, state *sriovnetworkv1.SriovNetworkNod
 	}
 	return false
 }
+
+func isSwitchDevModeEnabled(pciAddress string) (bool, error) {
+	glog.Infof("mellanox-plugin isSwitchDevModeEnabled(): device %s", pciAddress)
+	args := []string{"dev", "eswitch", "show", fmt.Sprintf("pci/%s", pciAddress)}
```

Comment:
please move this from the Mellanox to generic plugin. also please use netlink to get/set switchdev mode.
https://github.com/vishvananda/netlink/commit/fd97bf4e47867b5e794234baa6b8a7746135ec10